python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -167,6 +170,14 @@ def run_mtdome() -> None:
     asyncio.run(MTDomeCsc.amain(index=None))
 
 
+@dataclass
+class MoveAzCommandData:
+    """Class representing the data for a moveAz command."""
```

Comment:
Please document the units. This is especially important for the dome because it uses degrees in some places and radians in others.